src/config.tsx

```diff
@@ -60,9 +60,9 @@ export const projects: Project[] = [
   {
     id: 'keto',
     title: `${brandPrefix}Keto`,
-    descriptiveTitle: 'Permission and Role Management',
+    descriptiveTitle: 'Global access control',
     description:
-      'Access Control and Permission Management Server. Use best practices (RBAC, ABAC, ACL, ...) to secure your application.',
+      'Authorization Server based on Google Zanzibar, providing RBAC, ABAC and ACL',
```

Comment:
We end with a period everywhere else

```suggestion
      'Authorization Server based on Google Zanzibar, providing RBAC, ABAC and ACL.',
```

Reply:
The text here is not really true - we do not provide these APIs. You can implement them using Keto if you want, but it's not an RBAC server.

From GitHub:

> Global and consistent permission and authorization server with an easy and granular permission language and sub 10-millisecond latency. Based on Google Zanzibar. Ships gRPC and REST APIs. Written in Go.